### ISSUE TITLE
Folding without end delimiter

### ISSUE THREAD

**Issue by tpeacock19** [emacs-origami/origami.el#33]
I'm trying to incorporate this fork with [org-super-agenda](https://github.com/alphapapa/org-super-agenda) and I can get it to work by modifying it slightly to place the `org-super-agenda-header-separators` at the end of a super agenda group. However, ideally I would not have to change anything and could possibly just rely on the `org-super-agenda-header-prefix` as it worked with the old version of origami.

In short, is it possible for this to work without specifying an end delimiter?

**Comment by jcs090218** [comment]
Hi, in the old origami does not specify the org parser, so I guess it work by accident with their default behaviour! I have not really dig into the org file yet. Can you paste the current behaviour and expect behaviour comparison?
